src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -573,7 +573,6 @@ private ValueGetter<VBuffer<float>> MakeGetterInd(DataViewRow input, int iinfo)
                                 if (key >= (uint)size)
                                     continue;
                                 editor.Values[count] = 1;
-                                editor.Indices[count++] = slot * size + (int)key;
```

Comment:
test?

Reply:
It also appears that there are tests failing in CI with this change.

Maybe the right change is to use 

https://github.com/dotnet/machinelearning/blob/01a362b9a4a31a774f4f59bbe942d78240a69c7d/src/Microsoft.ML.Core/Data/VBufferEditor.cs#L42-L44